ext/pgsql/tests/pg_select_001.phpt

```diff
@@ -3,49 +3,55 @@ PostgreSQL pg_select() - basic test using schema
 --EXTENSIONS--
 pgsql
 --SKIPIF--
-<?php include("skipif.inc"); ?>
+<?php include("inc/skipif.inc"); ?>
 --FILE--
 <?php
 
-include('config.inc');
+include('inc/config.inc');
+$schema_name = 'schema_pg_select_001';
 
 $conn = pg_connect($conn_str);
 
-pg_query($conn, 'CREATE SCHEMA phptests');
+pg_query($conn, "CREATE SCHEMA {$schema_name}");
 
-pg_query($conn, 'CREATE TABLE phptests.foo (id INT, id2 INT)');
-pg_query($conn, 'INSERT INTO phptests.foo VALUES (1,2)');
-pg_query($conn, 'INSERT INTO phptests.foo VALUES (2,3)');
+pg_query($conn, "CREATE TABLE {$schema_name}.foo (id INT, id2 INT)");
+pg_query($conn, "INSERT INTO {$schema_name}.foo VALUES (1,2)");
+pg_query($conn, "INSERT INTO {$schema_name}.foo VALUES (2,3)");
 
-pg_query($conn, 'CREATE TABLE phptests.bar (id4 INT, id3 INT)');
-pg_query($conn, 'INSERT INTO phptests.bar VALUES (4,5)');
-pg_query($conn, 'INSERT INTO phptests.bar VALUES (6,7)');
+pg_query($conn, "CREATE TABLE {$schema_name}.bar (id4 INT, id3 INT)");
+pg_query($conn, "INSERT INTO {$schema_name}.bar VALUES (4,5)");
+pg_query($conn, "INSERT INTO {$schema_name}.bar VALUES (6,7)");
 
 /* Nonexistent table */
 var_dump(pg_select($conn, 'foo', array('id' => 1)));
 
 /* Existent column */
-var_dump(pg_select($conn, 'phptests.foo', array('id' => 1)));
+var_dump(pg_select($conn, "{$schema_name}.foo", array('id' => 1)));
 
 /* Testing with inexistent column */
-var_dump(pg_select($conn, 'phptests.bar', array('id' => 1)));
+var_dump(pg_select($conn, "{$schema_name}.bar", array('id' => 1)));
 
 /* Existent column */
-var_dump(pg_select($conn, 'phptests.bar', array('id4' => 4)));
+var_dump(pg_select($conn, "{$schema_name}.bar", array('id4' => 4)));
 
 /* Use a different result type */
-var_dump(pg_select($conn, 'phptests.bar', array('id4' => 4), 0, PGSQL_NUM));
+var_dump(pg_select($conn, "{$schema_name}.bar", array('id4' => 4), 0, PGSQL_NUM));
 
 /* Empty array */
-var_dump(pg_select($conn, 'phptests.bar', array()));
+var_dump(pg_select($conn, "{$schema_name}.bar", array()));
 
 /* No array */
-var_dump(pg_select($conn, 'phptests.bar'));
+var_dump(pg_select($conn, "{$schema_name}.bar"));
 
-pg_query($conn, 'DROP TABLE phptests.foo');
-pg_query($conn, 'DROP TABLE phptests.bar');
-pg_query($conn, 'DROP SCHEMA phptests');
+?>
+--CLEAN--
+<?php
+require_once('inc/config.inc');
+$schema_name = 'schema_pg_select_001';
+
+$conn = pg_connect($conn_str);
 
+pg_query($conn, "DROP SCHEMA IF EXISTS {$schema_name} CASCADE");
```

Comment:
Shouldn't we be dropping the 2 tables created in the test?

Reply:
Fixed in https://github.com/php/php-src/pull/12608/commits/c6ec16c7193cc1be42fa5cc7cf96f342527fdb26

In fact, the behavior will not change even if we do not modify it.
The `cascade` option will also drop tables that depend on the schema being dropped.

However, based on your point, I thought it would be easier to understand if I explicitly deleted it for the sake of clarity. Thank you!